pkg/cri/server/container_create_test.go

```diff
@@ -19,6 +19,7 @@ package server
 import (
 	"context"
 	"path/filepath"
+	goruntime "runtime"
```

Comment:
[Elsewhere in the codebase](https://github.com/containerd/containerd/blob/548d984f274f4ead06cdb9f3fdff3bfd388462eb/runtime/v2/binary.go#L24) this is aliased as "gruntime".